roles/rhol_crc/tasks/main.yml

```diff
@@ -137,3 +137,17 @@
 - name: Add crc kubeconfig
   when: cifmw_rhol_crc_creds | bool
   ansible.builtin.import_tasks: add_crc_creds.yml
+
+- name: Configure DNS in CRC VM
+  delegate_to: crc
```

Comment:
this should use the `ocps` group, otherwise it will fail with the reproducer when the crc vm is called `crc-0`

Reply:
We can keep `crc` since it's the default name, and leverage `hosts: crc, ocps` to cover both use-cases.